clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2197,6 +2198,7 @@ private synchronized Properties getVeniceWriterProperties(PushJobSetting pushJob
 
   private Properties createVeniceWriterProperties(String kafkaUrl, boolean sslToKafka) {
     Properties veniceWriterProperties = new Properties();
+    CollectionUtils.populateWithPassThroughConfigs(props, veniceWriterProperties::setProperty);
```

Comment:
Since there are no pass-through prefixes in this case, calling that method might be unnecessary.
Would it be simpler to just do the following?
```Java
veniceWriterProperties.putAll(props.getAsMap());
```

Reply:
We do not want to dump all the configs from prop to veniceWriterProperties, only those that have the newly defined passthrough prefix. populateWithPassThroughConfigs does that.

Reply:
I do not see any filtering based on the prefix for this case. Could you point me to what prefix is being used here?

Reply:
https://github.com/linkedin/venice/pull/1702/files#diff-5cfbe145e027e97151e0604aa02ef0cc2fa606b7369b24e09d80e4bdb3c8337dR56 
prefix as the value of key PASS_THROUGH_CONFIG_PREFIXES_LIST_KEY